Sources/BraintreeCard/BTCardError.swift

```diff
@@ -51,4 +51,10 @@ enum BTCardError: Error, CustomNSError, LocalizedError {
             return [NSLocalizedDescriptionKey: "Failed to fetch Braintree configuration."]
         }
     }
+
+    // MARK: - Equatable Conformance
+
+    public static func == (lhs: BTCardError, rhs: BTCardError) -> Bool {
+        lhs.errorCode == rhs.errorCode
+    }
```

Comment:
This is required for any of the enums with associated types. It it's able to resolve the equality of the error codes automatically.